tests/apollo/test_skvbc_dbsnapshot.py

```diff
@@ -934,7 +934,7 @@ async def test_restore_from_snapshot_of_other(self, bft_network, tracker):
 
         crashed_replica = list(bft_network.random_set_of_replicas(1, {initial_prim}))[0]
         bft_network.stop_replica(crashed_replica)
-        await skvbc.send_n_kvs_sequentially(DB_CHECKPOINT_WIN_SIZE)  # run till the next checkpoint
+        await skvbc.send_n_kvs_sequentially(DB_CHECKPOINT_WIN_SIZE + 100)  # run till the next checkpoint
```

Comment:
maybe int(DB_CHECKPOINT_WIN_SIZE * 1.5) so that if we change this constant the test remains valid?